src/gourmet/image_utils.py

```diff
@@ -87,6 +88,12 @@ def image_to_bytes(image: Image.Image) -> bytes:
     return ofi.getvalue()
 
 
+def load_pixbuf_from_resource(resource_name: str) -> Pixbuf:
+    data = _get_data('gourmet', f'data/images/{resource_name}')
```

Comment:
```suggestion
    data = get_data(op.join('gourmet',' data', 'images', resource_name)
```

Don't forget to add a test, however simple :)

Reply:
This may not be the final API, as I'm considering consolidating resource management into its down module.